std/algorithm/iteration.d

```diff
@@ -860,6 +860,7 @@ template each(alias pred = "a")
     import std.meta : AliasSeq;
     import std.traits : Parameters;
 
+private:
```

Comment:
I came to see `label:` as a near antipattern - it's too nonlocal and makes code difficult to understand and review without context. @wilzbach how about a rule that prevents at least the use of `private:` and `public:` in phobos?